src/e2e/puppeteer/__tests__/thought-trim.ts

```diff
@@ -0,0 +1,27 @@
+import click from '../helpers/click'
+import getEditingText from '../helpers/getEditingText'
+import paste from '../helpers/paste'
+import press from '../helpers/press'
+
+vi.setConfig({ testTimeout: 20000, hookTimeout: 20000 })
+
+it('when cursor is null, thought will be trimmed.', async () => {
+  const importText = `
+  -    Hello World.      `
+
+  await paste(importText)
+  await click('#content')
```

Comment:
Let's avoid tightly coupling this test to the unrelated `#content` element. Use `selection.clear` or activate cursorBack instead.